src/stores/settings.js

```diff
@@ -71,6 +73,10 @@ export const useSettingsStore = defineStore('settings', {
 			}
 			}
 		},
+
+		getBlurBackgroundEnabled: (state) => {
```

Comment:
getter makes no sense, If you don't use any logic here (like choose to support either via talk-config or via BrowserStorage here)

Reply:
true true